thats_the_way.py

```diff
@@ -0,0 +1,6 @@
+import os
+_path = input("enter your directory path")
```

Comment:
Nice work,
But please enter your code into a function, and remember to document it.